packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -131,10 +131,18 @@ async def subscribe(
         message_handler: MessageHandler,
         *,
         exclusive_queue: bool = True,
-    ) -> None:
+        topics: list[str] | None = None,
```

Comment:
Since you are suggestion to use `*.CRITICITY` to subscribe to a topic maybe add a type to enforce that topic should be composed of two words. Since users can use `WORD1.WORD2.CRITICTY`.

Otherwise I would propose to suggest subscribing with `#.CITICITY` to the topics, if I understand correctly this catches all for example, logs of log level `CRITICITY`.

Reply:
I am not sure I get what you mean. You can define as many topics as you wish, there are virtually no limits but the length of the routing key which is limited to 255 bytes.  so I do not understand why you want to fix it to 2 words.
I will maybe define a type that checks the max length and the used characters.